packages/sanity/src/structure/comments/src/components/list/CommentsListItemLayout.tsx

```diff
@@ -10,33 +10,25 @@ import {
   CommentDocument,
   CommentEditPayload,
   CommentMessage,
+  CommentReactionOption,
   CommentStatus,
   MentionOptionsHookValue,
 } from '../../types'
 import {FLEX_GAP} from '../constants'
 import {hasCommentMessageValue, useCommentHasChanged} from '../../helpers'
 import {AVATAR_HEIGHT, CommentsAvatar, SpacerAvatar} from '../avatars'
+import {CommentReactionsBar} from '../reactions'
 import {CommentsListItemContextMenu} from './CommentsListItemContextMenu'
 import {TimeAgoOpts, useTimeAgo, useUser, useDidUpdate} from 'sanity'
 
-const ContextMenuBox = styled(Box)``
-
-function StopPropagationWrap(props: React.PropsWithChildren<{isParent?: boolean}>) {
-  const {children, isParent} = props
-
-  const handleClick = useCallback((event: React.MouseEvent<HTMLDivElement>) => {
-    event.stopPropagation()
-  }, [])
+const stopPropagation = (e: React.MouseEvent<HTMLDivElement>) => e.stopPropagation()
 
-  return (
-    <ContextMenuBox data-root-menu={isParent ? 'true' : 'false'} onClick={handleClick}>
-      {children}
-    </ContextMenuBox>
-  )
-}
+const ContextMenuBox = styled(Box)``
```

Comment:
Do we need to continue defining this this way? Maybe just use box?

Reply:
Yes, it is actually needed, because we target the `ContextMenuBox` element in CSS like this:

```js
${ContextMenuBox} {
  &:focus-within {
    opacity: 1;
  }
}
```

Reply:
We could refactor this to use an data attribute instead, like:

```js

[data-ui="ContextMenuBox"] {
  &:focus-within {
    opacity: 1;
  }
}

```

Reply:
Missed that part! It's ok! Don't worry about updating it. Having the relation through the `ContentMenuBox` is less error prone, as removing it will show an error, but using the `data-ui` approach won't.
Thanks!